openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -288,9 +288,7 @@ data:
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-istio
                       - name: IMAGE_webhook
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-webhook
-                      image: quay.io/openshift-knative/knative-serving-operator:v0.7.1-TP1-03
-                      args:
-                        - --filename=https://raw.githubusercontent.com/openshift/knative-serving/release-v0.8.1/openshift/release/knative-serving-ci.yaml # remove this from individual release branches
```

Comment:
Do you really want to remove the --filename? I suppose then you won't be using image built in CI but already released images.

Reply:
1. We are not doing `sed` operation on `--filename` in e2e-tests-openshift.sh file like earlier we do 
2. here images have same name which was pushed by CI

Reply:
@mgencur images are replaced via the enviornment variables in the CS now.